site/en/tutorials/sequences/text_generation.ipynb

```diff
@@ -344,9 +346,15 @@
       "source": [
         "### Create training examples and targets\n",
         "\n",
-        "Divide the text into training examples and targets. Each training example will contain `seq_length` characters from the text. The corresponding targets contain the same length of text, except shifted one character to the right. For example, say `seq_length` is 4 and our text is \"Hello\", create one training example \"Hell\", and one target \"ello\".\n",
+        "Divide the text into training examples and targets. Each training example will contain `seq_length` characters from the text. \n",
         "\n",
-        "Break the text into chunks of `seq_length+1`:"
+        "Key Term: **Teacher forcing** - This example uses teacher forcing. During training the true sequence is input at each time step, not the model's output. \n",
```

Comment:
If this is a key term, this doesn't really explain it.
If it's important, maybe we should get it in the [MLCC glossary](https://developers.google.com/machine-learning/glossary/) and link off to it?